common/src/main/java/com/microsoft/identity/common/AndroidPlatformComponents.java

```diff
@@ -334,4 +344,34 @@ public void remove(String key) {
     public @NonNull IPlatformUtil getPlatformUtil() {
         return new AndroidPlatformUtil(mContext, mActivity);
     }
+
+    @Override
+    public IKeyStoreAccessor getKeyStore() {
```

Comment:
fyi: @shahzaibj, this means there'll be an extra work to get PRTv3 to work on linux. Either we implement this, or migrate this logic to use what we already have.